### PR TITLE
feat(interactions/application_commands): add support for channelTypes…

### LINKED ISSUE #232: Add channel type filter to command options
https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-option-structure

### PR DESCRIPTION
**Pull request by DjDeveloperr**
… in Option

fix #232

### CODE REVIEW

**Comment by DjDeveloperr** [comment]
@Helloyunho can you add `hacktoberfest-accepted` label after reviewing? 👀

File: src/interactions/applicationCommand.ts [hunk 1]
```diff
@@ -262,6 +263,16 @@ export function transformApplicationCommandOption(
   if (typeof data.options === 'object' && Array.isArray(data.options)) {
     data.options = data.options.map(transformApplicationCommandOption)
   }
+  if (
+    typeof data.channelTypes === 'object' &&
+    Array.isArray(data.channelTypes)
```
Comment (Helloyunho) [inline src/interactions/applicationCommand.ts:268]: maybe we can use `Array.isArray` only

**Review by Helloyunho — CHANGES_REQUESTED** [review]

File: src/interactions/applicationCommand.ts (hunk 1 above)
Comment (DjDeveloperr) [reply src/interactions/applicationCommand.ts]: done

**Review by Helloyunho — APPROVED** [review]
lgtm :imwave:

**Pull request merged**